### DISCUSSION TITLE
Can't get Gmail to authenticate, keeps rejecting correct username and password

### DISCUSSION THREAD

**Discussion by HowellBP** [gogs/gogs#7382] [Q&A]
* I'm on gogs 0.13.0
* I am using 2FA on my Gmail account
* I have enabled an app password, which works with several other apps (gogs is the only one that doesn't work)
* I have configured the app.ini to use email
* I have tried both 465 and 587
* I have tried with and without plaintext
* I have tried with and without single and double quotes

I get the exact same error message every time regardless of what configuration settings I use: `Failed to send test email to 'howellbp@gmail.com': gomail: could not send email 1: Auth: 535 5.7.8 Username and Password not accepted.`

What method does gogs/gomail use to send the username and password? I'm assuming whatever method it's currently using is not compatible with Google's current method of accepting username and password for SMTP.

**Comment by annpocoyo** [comment]
Is use less secure apps on. Google requires this to be on for smtp emails.
